README.rst

```diff
@@ -57,56 +42,43 @@ Here is a quick example of plotting an AIA image:
 
 .. code:: python
 
-    >>> import sunpy.map
-    >>> from sunpy.data.sample import AIA_171_IMAGE
-    >>> aia = sunpy.map.Map(AIA_171_IMAGE)
-    >>> aia.peek()
+   >>> import sunpy.map
+   >>> from sunpy.data.sample import AIA_171_IMAGE
+   >>>
+   >>> import matplotlib.pyplot as plt
+   >>>
+   >>> aia = sunpy.map.Map(AIA_171_IMAGE)
+   >>>
+   >>> aia.plot()
+   >>>
+   >>> plt.show()
```

Comment:
Why not just use `.peek()`?

Reply:
I thought the plan is to remove peek.

Reply:
I don't think so? I can't find an issue for it, and peek() does exactly what this example shows which is plot and show a figure in one line.

Reply:
For me peek is redundant API when plot and show do the same thing and give a user more control. 

A discussion to have on an issue. But we avoid using peek in our docs and gallery and the readme should follow the same.

Reply:
Ha, maybe I've changed my mind over the years without noticing! For a single plot that immediately shows (ie. by design the user can't modify it anyway, and Figure/Axes creation and deletion are done in that single blocking command) I think it's fine. I definitely don't want to hold up this PR though, so will resolve this conversation.